src/step_definitions/screenshots.ts

```diff
@@ -0,0 +1,33 @@
+import { Then } from 'cucumber';
+
+Then(/^I take screenshot of the element "([^"]*)" and save as a "([^"]*)"$/, async function(
```

Comment:
Should be `When` we do not validate anything here, we actually performing an action